app/views/foods/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<p style="color: green"><%= notice %></p>
+
+<h1 class="mx-3 ">Foods</h1>
+<div class="d-flex justify-content-end">
+  <%= link_to "Add Food", new_food_path, class: "btn btn-success mx-5 "  %>
+</div>
+
+<div id="foods">
+  <table class="table table-bordered m-3">
+    <tbody >
+      <tr>
+        <th class="text-primary display-6" colspan="6">Food</th>
+        <th class="text-primary " colspan="6">Measurement unit</th>
+        <th class="text-primary " colspan="6">Price</th>
+        <th class="text-primary " colspan="6">Quantity</th>
+        <th class="text-primary " colspan="6">Action</th>
+      </tr>
+      <% @foods.each do |food| %>
+        <tr>
+          <td colspan="6"><%= food.name %></td>
+          <td colspan="6"><%= food.measurement_unit %></td>
+          <td colspan="6">$ <%= food.price %></td>
+          <td colspan="6"><%= food.quantity %></td>
```

Comment:
- [ ] Impressive work so far in crafting such a stunning `UI`! 🌟 Your design showcases significant talent. Nevertheless, there are a few UI enhancements I'd recommend to ensure your design is perfectly aligned with the wireframe provided in the project requirements. 😊

   - Upon careful examination of the wireframe, you'll observe that it features just four `th` elements, specifically: `Food`, `Measurement unit`, `Price`, and `Action`. Regrettably, your current UI deviates from this layout as it incorporates an additional `quantify` column. This additional column isn't present in the wireframe and affects the alignment with the provided design.

   - To ensure your design accurately reflects the wireframe, I kindly suggest considering the removal of this extra `quantify` column from your UI. This adjustment will help your design peasing seamlessly with the wireframe, adhering to the project's requirements. 😊👍 See below for more details:

<table>
<tr>
<th align=“middle”>Wireframe</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="1345" alt="Screenshot 2023-10-28 at 05 00 41" src="https://github.com/kerlos-bahaa/recipe-app-/assets/107192076/8527c84b-3ab5-419a-bae1-5e012b4e6b78">
</td>
<td>
<img width="1345" alt="Screenshot 2023-10-28 at 05 00 41" src="https://github.com/kerlos-bahaa/recipe-app-/assets/107192076/637e8afd-191d-4e19-a73a-60ce431e6d83">
</td>
</tr>
</table>

Happy coding! 😊🚀👨‍💻